packages/editor/src/components/editor-help/index.native.js

```diff
@@ -76,16 +81,12 @@ function EditorHelpTopics( { close, isVisible, onClose } ) {
 					name="help-topics"
 				>
 					<View style={ styles.container }>
-						<BottomSheet.NavigationHeader
-							isFullscreen
-							leftButtonOnPress={ close }
-							leftButtonText={ __( 'Close' ) }
-							screen={
-								postType === 'page'
-									? __( 'How to edit your page' )
-									: __( 'How to edit your post' )
-							}
-						/>
+						<BottomSheet.Header>
+							<BottomSheet.Header.CloseButton onPress={ close } />
```

Comment:
As questioned elsewhere, I think we may be better off passing the desired text here. 

```suggestion
							<BottomSheet.Header.CancelButton onPress={ close } text={ __( 'Close' ) } />
```

Reply:
I think you mean to write 
```
<BottomSheet.Header.CancelButton onPress={ close } text={ __( 'Close' ) } />
```
Looking at the code I am not sure it is an improvement. Mostly since it creates confusion. 
Maybe we should change it to? 
```
<BottomSheet.Header.ReturnButton onPress={ close } text={ __( 'Close' ) } />
```

Reply:
> I think you mean to write
> 
> ```
> <BottomSheet.Header.CancelButton onPress={ close } text={ __( 'Close' ) } />
> ```

Ah, yes. I did make a typo with the `text` string. 

> Looking at the code I am not sure it is an improvement. Mostly since it creates confusion.
> Maybe we should change it to?
> 
> ```
> <BottomSheet.Header.ReturnButton onPress={ close } text={ __( 'Close' ) } />
> ```

Confusion between the "CancelButton" component name and "Close" text? Possibly. If you believe renaming `CancelButton` provides more clarity, that works for me. `ReturnButton` or `DismissButton` could potentially work.

Reply:
Leaving one additional note that this conversation will be moot if we decide to retain the `CloseButton`, which I am open to doing.